app/views/candidate_interface/other_qualifications/details/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title, title_with_error_prefix(t('page_titles.qualification_details'), @form.errors.any?) %>
-<% content_for :before_content, govuk_back_link_to %>
+<% content_for :before_content, govuk_back_link_to(refresh: true) %>
```

Comment:
Could we pass in the path to the previous page here, rather than relying on javascript?

Reply:
Yep that seems to work ok - have made the changes 👍